_docs/for-contributors/2017-01-01-building-from-source.md

```diff
@@ -39,7 +47,31 @@ Switch to the latest release branch, for example, `cardano-sl-1.0`:
 
     $ git checkout cardano-sl-1.0
 
-Then enter `nix-shell`:
+## Pure Nix build mode (recommended)
+
+Two simple steps:
```

Comment:
There should be a step how to install Nix via `curl https://nixos.org/nix/install | sh` and mention that Linux and Darwin are supported.

Reply:
Done.